build-bot-status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import sys
+import time
+import threading
+
+import requests
+
+from light_controller import LightController
+
+
+class BuildBotSession:
```

Comment:
the buildbot session should have 2 main components:
- BuildContextDAO
- LightController

- logging can be applied with decorator pattern with some important method